### ISSUE TITLE
menu in right sidebar is broken

### ISSUE THREAD

**Issue by viterbi-decoder** [microsoft/vscode#160055]
Does this issue occur when all extensions are disabled?: Yes

Version: 1.71.0 (user setup)
Commit: 784b0177c56c607789f9638da7b6bf3230d47a8c
Date: 2022-09-01T07:36:10.600Z
Electron: 19.0.12
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22000
Sandboxed: No

Steps to Reproduce:

Repro gif:

https://user-images.githubusercontent.com/111471419/188360240-049c8759-2cde-4558-a658-8f899459729a.mp4

**Comment by gjsjohnmurray** [comment]
By "broken" do you mean that it overflows the right edge of the screen?

**Comment by viterbi-decoder** [comment]
No, I mean when I click the menu, it immediately activated the first option when I release the cursor, but only on the right side.

**Comment by viterbi-decoder** [comment]
When the sidebar is on the right, and I click “Settings”, it immediately activates the “Check for updates” menu item, without any extra click.

**Comment by gjsjohnmurray** [comment]
I can repro this with `"window.titleBarStyle": "native"` and with enough space on the right of the window to make the context menu appear on the right.

It doesn't happen with `"window.titleBarStyle": "custom"` which is the default.
